### ISSUE TITLE
[FRAGMENTS] Confuding statement needs to be reworded....

### ISSUE THREAD

**Issue by mwherman2000** [w3c-ccg/did-spec#137]
In https://w3c-ccg.github.io/did-spec/#fragments, it states...

>A DID fragment MUST be used only as a method-independent pointer into the DID Document to identify a unique key description or other DID Document component. To resolve this pointer, the complete DID reference including the DID fragment MUST be used as the value of the id key for the target JSON object.

This is confuding.  It needs to be clarified and reworded.

**Comment by dmitrizagidulin** [comment]
Reworded in what sense? What would you like to see?

**Comment by mwherman2000** [comment]
Re-state this sentence/paragraph in plain English. I don't know what it is trying to say or what it is supposed to say ...to be able to be more clear with my feedback.

**Comment by talltree** [comment]
Michael: I am the author of those two sentences and, in the context of the
portion of the spec in which they appear, they are a straightforward a
statement of purpose. All of the terms it uses—DID fragment, DID method,
DID Document, target JSON object—are defined in the spec or surrounding
text.

If you take it out of context, the one term that might be confusing is
"unique key", because in the context of a DID document, that could be
interpreted as "a key in a JSON key-value pair" (which is what was
intended) or a "publicKey" property (which, ironically, can also have a
unique JSON key to look it up).

So I do suggest we fix that. What else about it is not clear to you?

**Comment by mwherman2000** [comment]
Here's a suggestion...

> A DID fragment MUST be used only as a method-independent reference into the DID Document to identify a component of a DID Document (e.g. a unique key description). To resolve this reference, the complete DID reference including the DID fragment MUST be used as the value of the key for the target component in the DID Document object.

**Comment by talltree** [comment]
Love it. Print!

**Comment by mwherman2000** [comment]
@jandrieu Can you make the above change to the spec-text?
